tests/integration/cattletest/core/test_local_auth.py

```diff
@@ -0,0 +1,187 @@
+from common_fixtures import *  # NOQA
+from requests.auth import AuthBase
+import requests
+from cattle import from_env
+
+
+class LocalAuth(AuthBase):
+    def __init__(self, jwt, prj_id=None):
+        # setup any auth-related data here
+        self.jwt = jwt
+        self.prj_id = prj_id
+
+    def __call__(self, r):
+        # modify and return the request
+        r.headers['Authorization'] = 'Bearer ' + self.jwt
+        if self.prj_id is not None:
+            r.headers['X-API-Project-Id'] = self.prj_id
+        return r
+
+
+def turn_on_off_local_auth(admin_client, request):
```

Comment:
Is it safe to do this given that we run tests in parallel and any test doing this could be running at the same time as any other test in the suite?